sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/SentryProperties.java

```diff
@@ -18,6 +19,9 @@
   /** Logging framework integration properties. */
   private @NotNull Logging logging = new Logging();
 
+  /** Report all or only uncaught web exceptions. */
+  private Integer exceptionResolverOrder = Ordered.HIGHEST_PRECEDENCE;
```

Comment:
`HIGHEST_PRECEDENCE` and also `int getOrder()` are primitive types, can we use `int` instead of `Integer`? just wondering if it'd be possible to be null or not

Reply:
Good point; changed it to `int`, just to be safe. @maciejwalkowiak suggested switching to a `boolean sentry.catch-unhandled-exceptions` above, so this might still change, especially in SentryProperties.